apps/fishing-map/features/footer/Footer.tsx

```diff
@@ -45,7 +45,7 @@ const FooterPartners = ({ smallScreen }: FooterPartnersProps) => {
           <a href="https://skytruth.org/" rel="noopener noreferrer" target="_blank">
             <img src={LogoSkytruth.src} alt="Skytruth" width="79px" height="24px" />
           </a>
-          <a href="https://google.com/" rel="noopener noreferrer" target="_blank">
+          <a href="https://google.com" rel="noopener noreferrer" target="_blank">
```

Comment:
Revert this change 🧹  if it's not needed

Reply:
I needed to have a change in the fishing map or the deploy is not working